packages/bpk-component-input/readme.md

```diff
@@ -22,30 +22,33 @@ export default () => (
     value="Edinburgh"
     onChange={() => console.log('input changed!')}
     placeholder="Country, city or airport"
+    clearButtonMode={CLEAR_BUTTON_MODES['while-editing']}
```

Comment:
This is strange `CLEAR_BUTTON_MODES['while-editing']`. I know we want the keys and values matched so that flow plays nicely. But i dont think we need to use a value that has a hyphen in it? Passing strings kinda sucks right?

Reply:
Hmm yeah true. I did it to match the React Native way, but I can camelCase it instead.

Reply:
Now changed!